src/bootstrap/defaults/bootstrap.compiler.toml

```diff
@@ -1,5 +1,7 @@
 # These defaults are meant for contributors to the compiler who do not modify codegen or LLVM
 [build]
+# Enforce stage0 to avoid checking stage1 std which requires to build stage1 compiler.
+check-stage = 0
```

Comment:
Question: should this have a changelog entry? Also, this is "default to stage 0" not "enforce stage 0" (since we don't error), right?